examples/flutter-ota/lib/main.dart

```diff
@@ -1,173 +1,12 @@
-    return Scaffold(
-      appBar: AppBar(
-        title: Text(Lt.of(context).list_title),
-      ),
-      body: ListView(
-        padding: const EdgeInsets.symmetric(vertical: 8.0),
-        children: _todos.map((Todo todo) {
-          return TodoItem(
-            todo: todo,
-            onTodoChanged: _handleTodoChange,
-          );
-        }).toList(),
-      ),
-      floatingActionButton: _isLoading
-          ? const CircularProgressIndicator()
-          : FloatingActionButton(
-              onPressed: () => _displayDialog(),
-              tooltip: Lt.of(context).addButton,
-              child: const Icon(Icons.add)),
-    );
-  }
-
-  void _handleTodoChange(Todo todo) {
-    setState(() {
-      todo.checked = !todo.checked;
-    });
-  }
-
-  void _addTodoItem(String name) {
-    setState(() {
-      _todos.add(Todo(name: name, checked: false));
-    });
-    _textFieldController.clear();
-  }
-
-  Future<void> _displayDialog() async {
-    return showDialog<void>(
-      context: context,
-      barrierDismissible: false, // user must tap button!
-      builder: (BuildContext context) {
-        return AlertDialog(
-          title: Text(Lt.of(context).title_addItem),
-          content: TextField(
-            controller: _textFieldController,
-            decoration: InputDecoration(hintText: Lt.of(context).hint_addItem),
-          ),
-          actions: <Widget>[
-            TextButton(
-              child: Text(Lt.of(context).button_addItem),
-              onPressed: () {
-                Navigator.of(context).pop();
-                _addTodoItem(_textFieldController.text);
-              },
-            ),
-          ],
-        );
-      },
-    );
-  }
-}
-
-class TodoApp extends StatelessWidget {
-  const TodoApp({super.key});
-
-  @override
-  Widget build(BuildContext context) {
-    return MaterialApp(
-      title: 'Todo list',
-      theme: ThemeData(
-        primarySwatch: Colors.blueGrey,
-      ),
-      onGenerateTitle: (context) => Lt.of(context).title,
-      localizationsDelegates: const [
-        Lt.delegate,
-        GlobalMaterialLocalizations.delegate,
-        GlobalWidgetsLocalizations.delegate,
-        GlobalCupertinoLocalizations.delegate,
-      ],
-      supportedLocales: Lt.supportedLocales,
-      home: const TodoList(),
-    );
-  }
-}
-
-// void main() {
-//   Lokalise.init('d24a6c4e93076b29e71347e8c39713f87f1e',
-//       '595412416388cd7cdebfe5.92023764');
-//   //Lokalise.setVersion(1);
-//   runApp(
-//     const TodoApp(),
-//   );
-// }
+import 'package:to_do_app/app/app.dart';
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   await Lokalise.init(
-      sdkToken: 'd24a6c4e93076b29e71347e8c39713f87f1e',
-      projectId: '595412416388cd7cdebfe5.92023764',
-      preRelease: false,
-      appVersion: 0);
-  runApp(const TodoApp());
+    sdkToken: 'd24a6c4e93076b29e71347e8c39713f87f1e',
+    projectId: '595412416388cd7cdebfe5.92023764',
```

Comment:
Should we include a real `sdkToken` and `projectId`? 

Maybe we should leave them empty and ask to fill them, what do you think?

Reply:
We should not include the token if it's actually used somewhere in Lokalise, I would say. If you have this token somewhere in your projects, I'd invalidate it

Reply:
I didn't include it, it was there I guess it was added by who did the first version of the sample app.

So do you think that we can just leave both empty with a comment requesting to fill them?